apps/crn-server/src/data-providers/contentful/teams.data-provider.ts

```diff
@@ -0,0 +1,240 @@
+import {
+  FetchOptions,
+  ListTeamDataObject,
+  TeamCreateDataObject,
+  TeamDataObject,
+  TeamTool,
+  TeamUpdateDataObject,
+} from '@asap-hub/model';
+
+import {
+  GraphQLClient,
+  FETCH_TEAMS,
+  FETCH_TEAM_BY_ID,
+  FetchTeamByIdQuery,
+  FetchTeamByIdQueryVariables,
+  FetchTeamsQuery,
+  FetchTeamsQueryVariables,
+  TeamsOrder,
+  Environment,
+  addLocaleToFields,
+} from '@asap-hub/contentful';
+
+import { TeamDataProvider } from '../teams.data-provider';
+
+export type TeamItem = NonNullable<
+  NonNullable<FetchTeamsQuery['teamsCollection']>['items'][number]
+>;
+
+type TeamFilter = {
+  active?: boolean;
+};
+
+export type FetchTeamsOptions = FetchOptions<TeamFilter>;
+
+const getCleanTools = (tools: TeamTool[]) =>
```

Comment:
this might sound like a silly requirement but maybe you can move all the helper functions below the class implementation? I think only the types need to be at the top